docs/sources/k6/next/javascript-api/k6/group.md

```diff
@@ -6,6 +6,12 @@ description: 'Runs code inside a group. Used to organize results in a test.'
 
 # group( name, fn )
 
+{{< admonition type="note" >}}
+
+For details about using the `check()` function with async values, refer to the jslib utils [check](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/jslib/utils/check/).
```

Comment:
Would it be better to link to the original `check` function here as well?

Reply:
There's a link to the original check function on the jslib utils check page, so I think just having the one link here is better. 🤓